src/widgets/allergies/allergy-form.test.tsx

```diff
@@ -9,7 +9,7 @@ import {
   mockSaveAllergyResponse,
   mockUpdatedAllergyResult
 } from "../../../__mocks__/allergies.mock";
-import { mockPatientId } from "../../../__mocks__/openmrs-esm-framework.mock";
+import { mockPatientId } from "../../setupTests";
```

Comment:
Ouch... No!

This file is used to initialize the tests. It should not export anything. It should not be used in imports. Its run by Jest implicitly.

If you have the structure we see above, its an indicator that you should not import anything, but declare that explicitly in the code.

Reply:
(You can also just do:

```diff
-import { mockPatientId } from "../../setupTests";
+import { mockPatient } from "../../../__mocks__/patient.mock";
+const mockPatientId = mockPatient.id;
```

)